src/GafferImageUI/ImageGadget.cpp

```diff
@@ -1012,20 +1691,20 @@ void ImageGadget::renderTiles() const
 
 			glBegin( GL_QUADS );
 
-				glTexCoord2f( uvBound.min.x, uvBound.min.y  );
-				glMultiTexCoord2f( GL_TEXTURE1, validBound.min.x, validBound.min.y  );
+				glTexCoord2f( uvBound.min.x, uvBound.min.y );
+				glMultiTexCoord2f( GL_TEXTURE1, validBound.min.x, validBound.min.y );
 				glVertex2f( validBound.min.x * pixelAspect, validBound.min.y );
 
-				glTexCoord2f( uvBound.min.x, uvBound.max.y  );
-				glMultiTexCoord2f( GL_TEXTURE1, validBound.min.x, validBound.max.y  );
+				glTexCoord2f( uvBound.min.x, uvBound.max.y );
+				glMultiTexCoord2f( GL_TEXTURE1, validBound.min.x, validBound.max.y );
 				glVertex2f( validBound.min.x * pixelAspect, validBound.max.y );
 
-				glTexCoord2f( uvBound.max.x, uvBound.max.y  );
-				glMultiTexCoord2f( GL_TEXTURE1, validBound.max.x, validBound.max.y  );
+				glTexCoord2f( uvBound.max.x, uvBound.max.y );
+				glMultiTexCoord2f( GL_TEXTURE1, validBound.max.x, validBound.max.y );
 				glVertex2f( validBound.max.x * pixelAspect, validBound.max.y );
 
-				glTexCoord2f( uvBound.max.x, uvBound.min.y  );
-				glMultiTexCoord2f( GL_TEXTURE1, validBound.max.x, validBound.min.y  );
+				glTexCoord2f( uvBound.max.x, uvBound.min.y );
+				glMultiTexCoord2f( GL_TEXTURE1, validBound.max.x, validBound.min.y );
```

Comment:
Especially for commits of this size, it's easier on the reader if unrelated formatting changes are split into a separate commit.

Reply:
Addressed in [5d14e21](https://github.com/GafferHQ/gaffer/pull/6329/commits/5d14e2131c0bce41f3dacce56d53ed3a8c81de11)